packages/c/carbon-charts.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "carbon-charts",
+  "description": "Carbon Charts component library",
+  "keywords": [
+		"charts",
+		"graphs",
+		"radar",
+		"gauge",
+		"donut",
+		"pie",
+		"sparkline",
+		"tree",
+		"treemap",
+		"heatmap",
+		"wordcloud",
+		"histogram",
+		"alluvial",
+		"geo",
+		"bar",
+		"bullet",
+		"scatter",
+		"meter",
+		"line",
+		"javascript",
+		"component",
+		"carbon",
+		"ibm",
+		"svg",
+		"data",
+		"typescript"
+  ],
+  "license": "Apache-2.0",
+  "homepage": "https://www.carbondesignsystem.com/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/carbon-design-system/carbon-charts/tree/master/packages/core"
+  },
+  "filename": "dist/umd/bundle.umd.js",
+  "autoupdate": {
+    "source": "npm",
+    "target": "@carbon/charts",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*.css",
+          "**/*.css.map"
```

```suggestion
          "*.css?(.map)"
```
You can combine this into a single glob, and I don't believe you need a glob star here, as the CSS files aren't in nested folders?